pgx-utils/src/sql_entity_graph/pg_extern/mod.rs

```diff
@@ -484,29 +488,33 @@ impl PgExtern {
                         let mut funcctx: ::pgx::PgBox<pg_sys::FuncCallContext>;
                         let mut iterator_holder: ::pgx::PgBox<IteratorHolder<#retval_ty_resolved>>;
 
-                        if ::pgx::srf_is_first_call(#fcinfo_ident) {
-                            funcctx = pgx::srf_first_call_init(#fcinfo_ident);
-                            funcctx.user_fctx = pgx::PgMemoryContexts::For(funcctx.multi_call_memory_ctx).palloc_struct::<IteratorHolder<#retval_ty_resolved>>() as *mut ::core::ffi::c_void;
+                        // SAFETY: I know what I'm doing. Scout's honor!
```

Comment:
```suggestion
```
I should at some point either give a real reason or I should add equally-flippant `// SAFETY:` remarks to other parts of the PGX expansions so that all of the functions we make pass relevant clippy lints, but I am deciding I should joke around slightly less at the moment and not add `// SAFETY` remarks if I am not going to explain them (so we deliberately don't pass the relevant clippy lints).